spring-cloud-generator/scripts/resources/googleapis_modification_string.txt

```diff
@@ -1,4 +1,4 @@
-_spring_cloud_generator_version = "4.2.1-SNAPSHOT" # {x-version-update:spring-cloud-gcp:current}
+_spring_cloud_generator_version = "4.3.1-SNAPSHOT" # {x-version-update:spring-cloud-gcp:current}
```

Comment:
Ah, this version didn't get updated with the annotation like the pom file did @emmileaf Note to self to follow-upon this (https://github.com/googleapis/release-please/blob/main/docs/customizing.md#updating-arbitrary-files)